soc/ambiq/apollo4x/Kconfig.defconfig

```diff
@@ -6,4 +6,18 @@ if SOC_SERIES_APOLLO4X
 
 rsource "Kconfig.defconfig.apollo4*"
 
+config PM
+	default y
+
+config PM_DEVICE
+	default y if PM
+
+config PM_DEVICE_RUNTIME
+	default y if PM_DEVICE
```

Comment:
can you explain why are you enabling these options? these are not needed for PM operations

Reply:
I removed them and will place them in the proj.conf file of whatever sample/test I am using if needed.